src/graph_compute/phase_scheduling.h

```diff
@@ -117,7 +128,7 @@ static void createChunkData(vertex_t * const nodes, const vid_t cntNodes,
   cilk_for (vid_t i = 0; i < scheddata->cntChunks; ++i) {
     chunkdata_t * chunk = &scheddata->chunkdata[i];
     chunk->nextIndex = i << CHUNK_BITS;
-    chunk->phaseEndIndex[0] = std::min(chunk->nextIndex + (1 << (CHUNK_BITS - 1)),
+    chunk->phaseEndIndex[0] = std::min(chunk->nextIndex + ((1 << CHUNK_BITS) >> 1),
```

Comment:
OK - I pushed the the changes to pr_rollback.  I even learned how to use stash to go back, apply this change, then rebase the numa_scheduling branch, and finally apply the stashed changes.  Of course, it was a pain in the ass and I only knew what was going on about 40% of the time, but it seems to have worked.  Anyway, I'll do a PR for numa_scheduling relative to master tomorrow sometime.